keyboards/royal_kludge/rk61/rgb/readme.md

```diff
@@ -0,0 +1,67 @@
+# RK61 RGB
+
+![RK61 RGB](https://i.pinimg.com/originals/01/e2/1f/01e21f1ba56526a0f2fe5047b6d25f11.png)
+
+Royal Kludge RK61 non-hotswap rgb edition, original PID 0x24F.
+
+Tested on the Royal Kludge RK61 non-hotswap RGB dual-mode edition, original PID 0x24F
+(RK61 firmware v1.03.000n, purchased after May 2020).
+
+- MCU: HFD2201KBA (rebrand SN32F248BF)
+- PCB: RK61-JX-RGB-HFD; REV: 01; 2019/08/13
+- Wired only, no bluetooth support.
+
+Keyboard Maintainer: [euwbah](https://github.com/euwbah)
```

Comment:
Add yourself here too if you want to become a maintainer for this keyboard.

Reply:
I've added myself as a keyboard maintainer.